### PR TITLE
Migrate to Yarn 2

### PR DESCRIPTION
**Pull request by djhi**
First part of the migration. Another PR will enable PnP if possible

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,10 @@ cypress/videos
 cypress/screenshots
 /public
 /storybook-static
+.pnp.*
+.yarn/*
```
Comment (fzaninotto) [inline .gitignore:20]: yet there is a file in the .yarn repository added in this PR

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (djhi) [reply .gitignore:20]: Yes, look at the following lines ;-)

Comment (djhi) [reply .gitignore:20]: I follower the yarn 2 documentation: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Comment (fzaninotto) [reply .gitignore:20]: right, thanks!

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**